src/uu/lsmem/src/lsmem.rs

```diff
@@ -475,9 +503,46 @@ fn print_table(lsmem: &Lsmem, opts: &Options) {
     table.with(Disable::column(ByColumnName::new("NODE")));
     table.with(Disable::column(ByColumnName::new("ZONES")));
 
+    if opts.noheadings {
+        table.with(Disable::row(Rows::first()));
+    }
+
     println!("{table}");
 }
 
+fn print_json(lsmem: &Lsmem, opts: &Options) {
+    let table_json = TableRowJson {
+        memory: create_table_rows(lsmem, opts),
+    };
+
+    let table_json_string = serde_json::to_string_pretty(&table_json).unwrap();
```

Comment:
I noticed that the generated json uses a string for the `removable` field whereas the original `lsmem` uses a bool.

Reply:
done